pkg/server/sticky_vfs.go

```diff
@@ -49,31 +39,18 @@ type StickyVFSRegistry interface {
 	// neither Open nor Get has been called with the given spec's StickyVFSID
 	// before, Open will create a new in-memory filesystem. Otherwise, the new
 	// Engine will re-open the existing store held within the in-memory
-	// filesystem.
-	//
-	// If the registry was created with the ReuseEnginesDeprecated option, Open
-	// will return an Engine whose Close method does not close the underlying
-	// engine. Subsequent Open calls will return the same Engine. With this
-	// option, at most one engine with a given id can be active at any given
-	// time. Note that with the ReuseEnginesDeprecated option, an Open call that
-	// returns an existing sticky engine will not respect modified configuration
-	// or attributes. When using the ReuseEnginesDeprecated option, the caller
-	// must call CloseAllEngines when they're finished to close the underlying
-	// engines.
+	// filesystem. The caller should ensure any previous engine accessing the
+	// same VFS has already been closed.
 	Open(ctx context.Context, cfg *Config, spec base.StoreSpec) (storage.Engine, error)
```

Comment:
Since the implementation for this method really only grabs the VFS from Get() and doesn't store any other state within the registry itself, thoughts on moving this out of the registry interface and just having a helper that takes a `StickyVFSRegistry`? Good with it either way, just throwing the idea